test/get-changelog-test.js

```diff
@@ -96,7 +99,7 @@ describe(getChangelog, function() {
       cwd: path.join(tmpPath, 'packages/my-app'),
     });
 
-    expect(changelog).to.include('[1.0.1]');
+    expect(changelog).to.include('[1.0.1](https://test.git.server:123/test-project/test-repo/compare/@scope/my-app@1.0.0...@scope/my-app@1.0.1)');
```

Comment:
Should these changes be applied to every test?

Reply:
Actually, lets make a separate test just for the repository stuff.

Reply:
added new test